cardinal/events/events.go

```diff
@@ -295,12 +265,11 @@ func WebSocketEchoHandler(ws *websocket.Conn) error {
 	}
 }
 
-func sendError(w http.ResponseWriter, erisError error) error {
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusInternalServerError)
-	err := json.NewEncoder(w).Encode(eris.ToJSON(erisError, true))
-	if err != nil {
-		return eris.Wrap(err, "")
+func FiberWebSocketUpgrader(c *fiber.Ctx) error {
```

Comment:
nit: I prefer `WebSocketUpgrader` to `FiberWebSocketUpgrader`

Reply:
nit: Also would be good to rename `WebSocketEchoHandler` to `WebSocketHandler` while we're at it?

Reply:
The echo handler is the term for echoing a message. I can get rid of Fiber, I don't care.

Reply:
Note the names that you are using on like 150 of `cardinal/server/server.go`, just a simple wsUpgrader and a wsHandler, I like those and think these fns should be the same imo

```
websocketUpgrader, websocketHandler := handler.WebSocketEvents(eventHub)
```

Reply:
websocketEchoHandler is different. The purpose is to handle a websocket connection by echoing an incoming message back to the sender. It's not used as part of cardinal. It was originally placed there as a test. It's used to test and maintain the semantics of the websocket.Connection api.